src/components/filters-editor.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import React, { useEffect, useState } from 'react';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import NativeSelect from '@material-ui/core/NativeSelect';
+import InputBase from '@material-ui/core/InputBase';
+import withStyles from '@material-ui/core/styles/withStyles';
+import { equipmentTypes } from '../utils/equipment-types';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+const useStyles = makeStyles(() => ({
+    root: {
+        flexGrow: 1,
+        paddingLeft: '40px',
+    },
+    textField: {
+        color: 'red',
+        width: '2033px',
```

Comment:
Why 2033px ?

Reply:
@ChamseddineBhd, @med-hedi is right, this is not a good practice to hardcode this. Is the app still "not so bad" responsive?